declarations/Cityscoot.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "Cityscoot",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.cityscoot.eu/paris/cgu",
+      "executeClientScripts": true,
+      "select": ".content"
```

Because `.content` is used on another element:
```suggestion
      "select": [
        "#cgu"
      ]
```